core-primitives/settings/src/lib.rs

```diff
@@ -104,4 +105,9 @@ pub mod node {
 	pub static RUNTIME_SPEC_VERSION: u32 = 3;
 	pub static RUNTIME_TRANSACTION_VERSION: u32 = 1;
 	pub static UNSHIELD: u8 = 6u8;
+
+	pub static TEERACLE_MODULE: u8 = 52u8;
+	pub static UPDATE_EXCHANGE_RATE: u8 = 0u8;
+	//send extrinsic to chain once per day
+	pub static MARKET_DATA_UPDATE_INTERVAL: Duration = Duration::from_millis(86400000);
```

Comment:
That might be silly of me.. but does it really make sense of have this duration specified in millis? 😋